freeipa/src/main/java/com/sequenceiq/freeipa/converter/cloud/InstanceMetaDataToCloudInstanceConverter.java

```diff
@@ -69,7 +71,13 @@ public CloudInstance convert(InstanceMetaData metaDataEntity) {
                 .orElseThrow(NotFoundException::new);
         Map<String, Object> cloudInstanceParameters = stackToCloudStackConverter.buildCloudInstanceParameters(stack.getEnvironmentCrn(), metaDataEntity);
         params.putAll(cloudInstanceParameters);
-        return new CloudInstance(metaDataEntity.getInstanceId(), instanceTemplate, instanceAuthentication, params);
+        return new CloudInstance(
+                metaDataEntity.getInstanceId(),
+                instanceTemplate,
+                instanceAuthentication,
+                metaDataEntity.getSubnetId(),
+                stack.getAvailabilityZone(),
```

Comment:
Note: As discussed, this will have to be replaced with `metaDataEntity.getAvailabilityZone()` once the respective change is made to `InstanceMetaData` (freeipa) in a later PR. Nothing to do now.